plugins/woocommerce/tests/php/src/Internal/Orders/OrderStatusRestControllerTest.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
```

Comment:
```suggestion
<?php
declare( strict_types = 1 );

```

Required by phpcs linter rule.